src/main/java/pl/kurs/models/WaterContainers.java

```diff
@@ -0,0 +1,47 @@
+package pl.kurs.models;
+
+public class WaterContainers {
+
+    private String name;
+    private double maxCapCm3;
+    private double currentFillingCm3;
+
+    public WaterContainers(String name, int maxCapCm3, int currentFillingCm3) {
```

Comment:
Make this constructor private and creat static factory method with arguments validation